.github/PULL_REQUEST_TEMPLATE/bug.md

```diff
@@ -0,0 +1,15 @@
+# Problem
+
+Describe the nature of the problem.
+
+# Solution
+
+Describe the fix and go into as much technical detail needed to help another developer understand.
+
+# Testing/Validation
+
+Enumerate on steps to validate that this is fixed.

```

Comment:
I like this! Particularly for when we get into production it will be good to document how we are ensuring the fix is appropriate (maximally effective and minimally invasive).